src/sms-gateway/ViewSmsGatewayEdit/ViewSmsGatewayEdit.js

```diff
@@ -47,16 +47,11 @@ const getFormComponent = (gatewayType) => {
     throw new Error(`The gateway type does not exist, got "${gatewayType}"`)
 }
 
-const getInitialValues = (gateway) => {
-    if (gateway.type === BULK_SMS_FORM) {
-        return {
-            ...gateway,
-            [FIELD_PASSWORD_CONFIRMATION_NAME]: gateway[FIELD_PASSWORD_NAME],
-        }
-    }
-
-    return gateway
-}
+const getInitialValues = (gateway) => ({
+    ...gateway,
+    password: null,
+    authToken: null,
```

Comment:
these are nulled out for the moment, though in the future the api should not return values for these properties.